src/PanZoom.js

```diff
@@ -556,7 +556,7 @@ class PanZoom extends React.Component<Props, State> {
     return dragContainer
   }
 
-  autoCenter = (zoomLevel: number = 1) => {
+  autoCenter = (zoomLevel: number = 1, withoutAnimation: boolean = false) => {
```

Comment:
I think it would be easier to understand by not using a negation here. Use `animate: boolean = true` instead

Reply:
done